### PR TITLE
Load benchmark data

### PR DESCRIPTION
**Pull request by EricHenry**
Set up home page to request benchmarks data and load it into the model.

### CODE REVIEW

File: src/Pages/Home_.elm [hunk 1]
```diff
@@ -1,11 +1,143 @@
...
-    , element = text "Hello, world!"
+    , element = text "Hello, Roc world!"
     }
+
+
+
+-- HTTP
+
+
+type alias Benchmark =
```
Comment (rvcas) [inline src/Pages/Home_.elm:86]: this is already defined

Comment (rvcas) [reply src/Pages/Home_.elm:86]: https://github.com/rvcas/roc-perf-stats/blob/main/src/Api/Benchmark.elm

File: src/Pages/Home_.elm [hunk 2]
```diff
@@ -1,11 +1,143 @@
...
+        }
+
+
+benchmarksDecoder : Decoder (List Benchmark)
+benchmarksDecoder =
+    Decode.list benchmarkDecoder
+
+
+benchmarkDecoder : Decoder Benchmark
+benchmarkDecoder =
```
Comment (rvcas) [inline src/Pages/Home_.elm:127]: can we move all this logic to the Api folder?

Comment (rvcas) [reply src/Pages/Home_.elm:127]: the benchmark file should be fine for this.

here is a good example using elm-spa: https://github.com/ryannhg/elm-spa-realworld/blob/master/src/Api/Article.elm

also notice how the `Failed, Loading, Success` concept is made into it's own data type in `Api.Data`

**Comment by EricHenry** [comment]
thanks for the feedback @rvcas ! let me know if I can make any other improvements 👍

File: src/Pages/Home_.elm [hunk 3]
```diff
@@ -1,11 +1,73 @@
...
+
+
+type alias Model =
+    Data (List Benchmark)
+
+
+init : ( Model, Cmd Msg )
+init =
+    ( Api.Data.Loading
+    , Api.Benchmark.getAll GotBenchmarks
```
Comment (rvcas) [inline src/Pages/Home_.elm:37]: is this a typo or intentionally past tense? `Get` vs. `Got`?

Comment (EricHenry) [reply src/Pages/Home_.elm:37]: It's intentional. 😅  I was trying to capture that this message means we received data for benchmarks. 

Similarly to the elm-spa example you sent me earlier ref: https://github.com/ryannhg/elm-spa-realworld/blob/master/src/Pages/Article/Slug_.elm#L71

**Review by rvcas — APPROVED** [review]
Looks great. Thank you. Merge whenever you are ready

**Pull request merged**